lib/python/vdm/server/HTTPListener.py

```diff
@@ -1463,7 +1463,7 @@ def get(database_id):
             return make_response(jsonify({'error': 'Not found'}), 404)
         else:
             if len(database['members']) == 0:
-                return jsonify({'status': 'errorNoMembers'})
+                return jsonify({"status": 200, "statusString": "OK", 'error': 'errorNoMembers'})
```

Comment:
It is not an error to have no members in a database. That is how it will be when you first create a database, correct? 
Also, the status and statusString properties are to indicate what happened with the request. If there is an error, status should use a code indicating that and statusString should have a string describing the error. If they are always 200 and OK, they are useless.